src/mgmt/srv_target.c

```diff
@@ -570,6 +570,7 @@ recreate_pooltgts()
 		}
 
 		D_INFO("recreating files for pool "DF_UUID"\n", DP_UUID(pool_info->spi_id));
+		rdb_blob_sz = 0;
 		rc = smd_rdb_get_blob_sz(pool_info->spi_id, &rdb_blob_sz);
```

Comment:
I do not understand your concern. From what I can tell, `smd_rdb_get_blob_sz()` disregards the value of `rdb_blob_sz` and  simply overwrites it. What is the point?